Remotion/Web/Core/Res/Themes/NovaGray/HTML/StyleUtility.js

```diff
@@ -27,23 +27,23 @@ StyleUtility.CreateBorderSpans = function (selector)
 StyleUtility.AddBrowserSwitch = function ()
 {
   var browser;
-  if (navigator.appVersion.indexOf ('Edge') !== -1)
+  if ((navigator.userAgent.indexOf ('Edge') !== -1) || (navigator.userAgent.indexOf ('Edg') !== -1))
   {
     browser = 'edge';
   }
```

Comment:
After talking with mike we concluded to drop support for the legacy Edge. The new edge is then mapped to `blink` (Chromium engine).

```suggestion
  if (navigator.userAgent.indexOf ('Edg/') !== -1)
  {
    browser = 'blink';
  }
```